src/app/utility/help-panel/help-panel.component.ts

```diff
@@ -10,23 +10,33 @@ import { IdbFacility } from 'src/app/models/idb';
 })
 export class HelpPanelComponent implements OnInit {
 
-
-  selectedFacility: IdbFacility;
+  helpText: string;
+  selectedSource: string;
   selectedFacilitySub: Subscription;
-  constructor(private facilityDbService: FacilitydbService) { }
+  selectedFacilityName: string = 'Facility';
+  
+  constructor(
+    private router: Router,
+    private facilityDbService: FacilitydbService,
+    ) { 
+    router.events.subscribe((route) => {
```

Comment:
The help ends up blank right away if you don't navigate to the component. (on refresh)

in ngOnInit() the same logic can be used with this.router.snapshot.url (it doesn't update on events so will still need this). I'd create a function that takes the url and sets these two properties.